src/components/views/rooms/RoomListHeader.tsx

```diff
@@ -261,11 +265,10 @@ const RoomListHeader = ({ onVisibilityChange }: IProps) => {
                         showCreateNewSubspace(activeSpace);
                         closePlusMenu();
                     }}
-                    disabled={!canAddRooms}
```

Comment:
We should keep the component if the user doesn't have permissions to create a space. On the other hand we can hide it if `shouldShowComponent(UIComponent.CreateSpaces)` is `true`.

Reply:
This is what I did (hopefully it's what you meant :) )
 - if the user cannot create spaces anywhere, hide the button completely.
 - if the user can create spaces, but cannot create subspaces in this particular space, show a disabled button.

Reply:
Never mind. I got the GitHub colours wrong.